components/empty-state/empty-state-simple-button.js

```diff
@@ -0,0 +1,75 @@
+import '../button/button-subtle.js';
+import { emptyStateSimpleStyles, emptyStateStyles } from './empty-state-styles.js';
+import { html, LitElement, nothing } from 'lit';
+import { bodyCompactStyles } from '../typography/styles.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+/**
+ * The `d2l-empty-state-simple-button` component is an empty state component that displays a description and action button.
+ * @fires d2l-empty-state-action - Dispatched when the action button is clicked
+ */
+class EmptyStateSimpleButton extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * Optional: The action text to be used in the subtle button
+			 * @type {string}
+			 */
+			actionText: { type: String, attribute: 'action-text' },
+			/**
+			 * REQUIRED: A description giving details about the empty state
+			 * @type {string}
+			 */
+			description: { type: String },
+		};
+	}
+
+	static get styles() {
+		return [bodyCompactStyles, emptyStateStyles, emptyStateSimpleStyles];
+	}
+
+	constructor() {
+		super();
+		this._missingDescriptionErrorHasBeenThrown =  false;
+		this._validatingDescriptionTimeout = null;
+	}
+
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+		this._validateDescription();
+	}
+
+	render() {
+		const actionButton = this.actionText
+			? html`<d2l-button-subtle class="d2l-empty-state-action" @click=${this._handleActionClick}  h-align="text" text=${this.actionText} slim></d2l-button-subtle>`
```

Comment:
Thanks, I made this change. Similarly, I also realized that most of my empty state component calls suffered from this same issue and would often span 150-200 characters for a line so I tried to split most of those calls up on to multiple lines for readability.

https://github.com/BrightspaceUI/core/blob/f28cd191df2a8b999d86195b7761b21b50ad0e43/components/empty-state/README.md?plain=1#L101

I thought this would also make the documentation more readable but I am unsure if this would appear as expected in the Daylight code blocks. It didn't seem to render properly on the demo page which is why I did not make those changes there.

Reply:
Thanks for checking!